### DISCUSSION TITLE
PFD stuck on rose display

### DISCUSSION THREAD

**Discussion by noelchr** [flybywiresim/aircraft#3134] [Help] [answered]
It appears that my PFD is displaying in rose mode regardless which position the EFIS control knob is in. Also the co-pilot EFIS is all white. I am running the updated sim version 1.7.8. I tried deleting the A32NX folder and then completely re-installing but it still results in the same issue. Any help ideas. Thanks!

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
try deleting all mods except for the a32nx

**Reply by noelchr** [reply]
Perfect! Thanks that worked.

**Comment by hotshotp** [comment]
this is a result of a corrupt installation usually, strange that it doesn't help you
